typescript-advanced/tasks/005-x-is-of-type-unknown.problem.ts

```diff
@@ -6,6 +7,19 @@ const somethingDangerous = () => {
 
 try {
   somethingDangerous();
-} catch (error) {
+} catch (error: any) {
```

Comment:
Also another way to fix is:
try {
  somethingDangerous();
} catch (error) {
  if (typeof error === "object" && error && "message" in error) {
    console.log(error.message);
  } else {
    throw error;
  }
}